core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/textannotation/Constituent.java

```diff
@@ -45,6 +47,7 @@
 
     protected final String viewName;
     protected Map<String, String> attributes;
+    protected Map<String, Double> labelsToScores;
```

Comment:
For Constituents and Relations, we should also update the serialization to JSON code to save the labelToScore field.